### PR TITLE
Adding Microconfig

### PR DESCRIPTION
**Pull request by kapodes**
Nobody is thinking about configuration management. Time to change that :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +73,10 @@ Please take a quick gander at the **[contribution guidelines](https://github.com
 - [spark](https://github.com/apache-spark-on-k8s/spark) - Apache Spark enhanced with native Kubernetes scheduler back-end.
 - [wallaroo](https://github.com/WallarooLabs/wallaroo) - Ultrafast and elastic data processing [https://www.wallaroolabs.com](https://www.wallaroolabs.com/)
 
+### Configuration
+- [microconfig](https://microconfig.io) - Modern and simple way to manage all types of configuration.
```
Comment (rootsongjc) [inline README.md:77]: Please use the GitHub repo link instead of the website.

**Comment by rootsongjc** [comment]
I think it will be better to place it into the tools category.

**Comment by kapodes** [comment]
Moved to Application Delivery since that's where it's usually used with all other tools in that section.

**Comment by rootsongjc** [comment]
Merged and published at https://jimmysong.io/awesome-cloud-native

**Pull request merged**